docs/switching_from/poetry.md

```diff
@@ -0,0 +1,32 @@
+# Transitioning from `poetry` to `pixi`
+Welcome to the guide designed to ease your transition from `poetry` to `pixi`.
+This document compares key commands and concepts between these tools, highlighting `pixi`'s unique approach to managing environments and packages.
+With `pixi`, you'll experience a project-based workflow similar to `poetry` while including the `conda` ecosystem and allowing for easy sharing of your work.
+
+## Why Pixi?
+Poetry is most-likely the closed tool to pixi in terms of project management, in the python ecosystem.
+`pixi` however adds the power of the conda ecosystem, allowing for a more flexible and powerful environment management.
+Not limited but including the PyPI ecosystem.
+
+## Quick look at the differences
+| Task                       | Poetry                                                            | Pixi                                                                                                                                              |
+|----------------------------|-------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| Creating an Environment    | `poetry new myenv`                                                | `pixi init myenv`                                                                                                                                 |
+| Running a Task             | `poetry run which python`                                         | `pixi run which python` `pixi` uses a build-in cross platform shell for run where poetry uses your shell.                                         |
+| Installing a Package       | `poetry add numpy`                                                | `pixi add numpy` adds the conda variant. `pixi add --pypi numpy` adds the PyPI variant of the package.                                            |
+| Uninstalling a Package     | `poetry remove numpy`                                             | `pixi remove numpy`                                                                                                                               |
```

Comment:
Is there a PyPI version for `pixi remove`? If yes, maybe mention it as well.

Reply:
Yep added it, thanks.